.github/workflows/build_wheels.yml

```
@@ -34,7 +34,7 @@ jobs:
           submodules: "recursive"
 
       - name: Build wheels
-        uses: joerick/cibuildwheel@v1.10.0
+        uses: joerick/cibuildwheel@v1.11.1
```

This repo recently migrated to the `pypa` namespace.
```suggestion
        uses: pypa/cibuildwheel@v1.11.1
```